libs/shared-atomic-design-components/src/lib/atoms/FormTextArea.tsx

```diff
@@ -31,24 +30,49 @@ function FormTextArea(props: any) {
     touched,
     errors,
     disabled,
+    maxLength,
   } = props
 
-  const hasError = !!get(touched, name) && !!get(errors, name)
+  const charactersLength = values[name]?.length || 0
+
+  const numberOfCharsRequiredToMinAmount =
+    minChar - (values[name] ? charactersLength : 0)
+
+  const numberOfCharsAllowedToMaxAmount =
+    maxChar - (values[name] ? charactersLength : 0)
+
+  const minCharAmountReached =
```

Comment:
Suggest to prefix with `is`. That makes it even more clear that this is a boolean variable.

Reply:
... and right below here on lines 47 and 49 you see a good rule applied: prefix all boolean variables with `is` or `has`. I've seen that advice repeated often in my career.